db/raw.go

```diff
@@ -8,11 +8,11 @@ import (
 )
 
 func Exec(ctx context.Context, client Client, query string, args ...any) (sql.Result, error) {
-	return client.Exec(ctx, query, args)
+	return client.Exec(ctx, query, args...)
 }
 
-func RawQuery[T any](ctx context.Context, client Client, query string, args ...any) (ts []T, err error) {
```

Comment:
I think leaving it as raw query will be more explicit in reading the code when using it elsewhere

Reply:
@khaicm-0897 The old Query struct is renamed to `QueryBuilder` that can be used in both read and mutation operations. There is no need to use `RawQuery`